spec/controllers/miq_ae_class_controller_spec.rb

```diff
@@ -63,10 +63,10 @@
   context "#domains_priority_edit" do
     it "sets priority of domains" do
       set_user_privileges
-      FactoryGirl.create(:miq_ae_namespace, :name => "test1", :parent => nil, :priority => 1)
-      FactoryGirl.create(:miq_ae_namespace, :name => "test2", :parent => nil, :priority => 2)
-      FactoryGirl.create(:miq_ae_namespace, :name => "test3", :parent => nil, :priority => 3)
-      FactoryGirl.create(:miq_ae_namespace, :name => "test4", :parent => nil, :priority => 4)
+      FactoryGirl.create(:miq_ae_domain, :name => "test1", :parent => nil, :priority => 1)
+      FactoryGirl.create(:miq_ae_domain, :name => "test2", :parent => nil, :priority => 2)
```

Comment:
@kbrock 
In the Factory we set the tenant if not specified to be the root_tenant